app/controllers/report_controller/reports/editor.rb

```diff
@@ -1661,7 +1661,7 @@ def build_selected_fields(rpt)
       f_len = fields.length
       for f_idx in 1..f_len # Go thru fields in reverse
         f_key = fields[f_len - f_idx].last
-        next if f_key.ends_with?(*CHARGEBACK_ALLOWED_FIELD_SUFFIXES)
+        next if f_key.ends_with?(*CHARGEBACK_ALLOWED_FIELD_SUFFIXES) || f_key.include?('managed')
```

Comment:
@zeari I suppose `managed` is not a suffix and we can't just extend the constant here. Too bad I liked to keep this simple.